backend/test/features/activity-delete.feature

```diff
@@ -5,51 +5,42 @@ Feature: Delete an object
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug          |
-      | bernd-das-brot|
-    And I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+      | spongebob     |
+    And we have the following post in our database:
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T23:11:16Z" title="Tuesday, May 7th 2019, 1:11:16 am +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,51 +5,42 @@ Feature: Delete an object
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug          |
-      | bernd-das-brot|
-    And I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+      | spongebob     |
+    And I send a POST request with the following activity to "/inbox":
```


```suggestion
    And we have the following posts in our database:
```
Create a separate step definition for test setup. In this scenario I think only the `id` is relevant, no? Maybe content, too, but certainly not the entire activity json.